front/shared/src/main/scala/com/kubukoz/lame/Client.scala

```diff
@@ -21,17 +21,17 @@ import org.http4s.dom.FetchClientBuilder
 import org.http4s.implicits._
 import smithy4s.http4s.SimpleRestJsonBuilder
 import org.http4s.Uri
-import org.scalajs.dom.HTMLElement
 import calico.IOWebApp
+import fs2.dom.HtmlElement
 
 object Client extends IOWebApp {
 
-  def render: Resource[IO, HTMLElement] = FetchClientBuilder[IO]
+  def render: Resource[IO, HtmlElement[cats.effect.IO]] = FetchClientBuilder[IO]
     .resource
     .flatMap { fetchClient =>
       SimpleRestJsonBuilder(HelloService)
         .client(fetchClient)
-        .uri(Uri.unsafeFromString(org.scalajs.dom.window.location.origin.get))
+        .uri(Uri.unsafeFromString(org.scalajs.dom.window.location.origin))
```

Comment:
if you care there is an effectual `window.location` now, see example at bottom of https://armanbilge.github.io/calico/iowebapp.html

Reply:
niice